pom.xml

```diff
@@ -59,7 +59,21 @@
         <carbon.registry.version.4.4.5>4.4.5</carbon.registry.version.4.4.5>        
         <carbon.rule.version.4.4.0>4.4.0</carbon.rule.version.4.4.0>
         <cipher.tool.version.1.0.0v3>1.0.0-wso2v3</cipher.tool.version.1.0.0v3>  
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>  
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <jaggery.version.0.10.1>0.10.1</jaggery.version.0.10.1>
+        <carbon.registry.version.4.4.5>4.4.5</carbon.registry.version.4.4.5>
+        <jaggery.modules.version.1.3.0>1.3.0</jaggery.modules.version.1.3.0>
+        <carbon.multitenancy.version.4.4.1>4.4.1</carbon.multitenancy.version.4.4.1>
+        <carbon.commons.version.4.4.4>4.4.4</carbon.commons.version.4.4.4>
+        <carbon.analytics-common.version.5.0.0-SNAPSHOT>5.0.0-SNAPSHOT</carbon.analytics-common.version.5.0.0-SNAPSHOT>
+        <carbon.identity.version.4.5.4>4.5.4</carbon.identity.version.4.5.4>
+        <carbon.event-processing.version.2.0.4-SNAPSHOT>2.0.4-SNAPSHOT</carbon.event-processing.version.2.0.4-SNAPSHOT>
+        <carbon.dashboards.version.1.0.1-SNAPSHOT>1.0.1-SNAPSHOT</carbon.dashboards.version.1.0.1-SNAPSHOT>
+        <cep.version.4.0.0-SNAPSHOT>4.0.0-SNAPSHOT</cep.version.4.0.0-SNAPSHOT>
```

Comment:
@suhothayan, will send the pull request when those are released. Sorry for the inconvenience caused.